test/contracts/ERC20WithoutReturningValueOnTransfer.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.24;
+
+import "openzeppelin-eth/contracts/token/ERC20/ERC20.sol";
+
+contract ERC20WithoutReturningValueOnTransfer is ERC20 {
+    function mint(uint256 amount, address beneficiary) public {
+        _mint(beneficiary, amount);
+    }
+
+    function setBalance(uint256 amount) public {
+        _mint(msg.sender, amount);
+    }
+
+    function transfer(address _to, uint256 _value) public returns (bool) {
+        super.transfer(_to, _value);
+    }
+       
+    function transferFrom(address _from, address _to, uint256 _value)
+    public returns (bool)
```

Comment:
Shouldn't we remove `returns (bool)` from the signature?

Reply:
If I define

`function transfer(address to, uint256 value) public`
`function transfer(address to, uint256 value) public returns (bool)`

It uses the first 4 butes of the result of `sha3('transfer(address, uint256)')` (function selector) where for both definitions are the same cause doesnt care about returns values nor visibility

But if I have:

`function transfer(address to, uint256 value) public`
`function transfer(address to, uint8 value) public`

Wont work cause the selectors are differents

E.G:

```Interfaz ERC20 {
   function transfer(address to, uint256 value) public returns (bool);
}

MyContract { 
     bool res = ERC20(token).transfer()
}```

Token contract

```Token {
function transfer(address to, uint256 value) public {
  ....
}```

The call will be success